### PR TITLE
Allow upgrading to 3.0

### PR DESCRIPTION
**Pull request by hpidcock**
Since #14298 has landed in 3.0, it is now possible to upgrade to 3.0.

## QA steps

Test an upgrade to 3.0

## Documentation changes

N/A

## Bug reference

N/A

### CODE REVIEW

**Review by juanmanuel-tirado — APPROVED** [review]
LG2M

File: apiserver/facades/client/client/client_test.go [hunk 1]
```diff
@@ -210,14 +210,10 @@ func (s *serverSuite) TestSetModelAgentVersionOldModels(c *gc.C) {
 	err := s.State.SetModelAgentVersion(version.MustParse("2.8.0"), nil, false)
 	c.Assert(err, jc.ErrorIsNil)
 	args := params.SetModelAgentVersion{
-		Version: version.MustParse(fmt.Sprintf("%d.0.0", jujuversion.Current.Major+1)),
```
Comment (ycliuhw) [inline apiserver/facades/client/client/client_test.go:213]: This test could be changed to 

```go
args := params.SetModelAgentVersion{
		Version: version.MustParse("3.0.0"),
}
c.Assert(err, gc.ErrorMatches, `
    these models must first be upgraded to at least 2.9.33 before upgrading the controller:
-admin/controller`[1:])
```

**Comment by barrettj12** [comment]
Awesome, once we get this into develop, we can finally fix the GitHub upgrade tests

**Pull request merged**